### ISSUE TITLE
Open in external editor fails

### ISSUE THREAD

**Issue by yingzhu146** [joouha/euporie#20]
When pressing `e` I'm getting the following warning, and the process never completes.

```python
/1ib/python3.9/site-packages/euporie/commands/base.py:165: RuntimeWarning: coroutine 'edit_in_external_editor' was never awaited
self.handler()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```

euporie 1.3.2, OSX,  export EDITOR=nvim

**Comment by joouha** [comment]
Hi!

I've fixed this in the dev branch here: https://github.com/joouha/euporie/commit/a740a7e6bbeeb912158d60fbc26f589cccdf8933

I'll make a new release tomorrow which includes this change

**Comment by joouha** [comment]
I've released [v1.4.0](https://github.com/joouha/euporie/releases/tag/v1.4.0) with the fix :+1:
